DirectProgramming/DPC++FPGA/Tutorials/Features/experimental/ip_authoring_mmhost/src/mmhost.cpp

```diff
@@ -0,0 +1,133 @@
+#include <CL/sycl.hpp>
+#include <iomanip>
+#include <sycl/ext/intel/experimental/interfaces.hpp>
+#include <sycl/ext/intel/fpga_extensions.hpp>
+
+#include "dpc_common.hpp"
+
+using namespace cl::sycl;
```

Comment:
Please remove this `using namespace cl::sycl;` and write the namespace in the following code